backend/src/database/migrations/20221114095422_create-server.js

```diff
@@ -3,6 +3,7 @@ export async function up(knex) {
     table.uuid('id').primary().defaultTo(knex.raw('(UUID())'));
     table.string('name', 255).notNullable();
     table.string('ipAddress', 255).notNullable();
+    table.string('email', 255).notNullable();
```

Comment:
Remove this.

Reply:
Changes have been made, email field removed.